src/seedu/addressbook/commands/Command.java

```diff
@@ -39,13 +39,15 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     /**
      * Executes the command and returns the result.
      */
-    public CommandResult execute(){
-        throw new UnsupportedOperationException("This method should be implement in child classes");
-    }
+//    public CommandResult execute(){
+//        throw new UnsupportedOperationException("This method should be implement in child classes");
+//    }
```

Comment:
Don't leave commented out code. Just go ahead and delete removed code, as we can recover them later